src/components/map.js

```diff
@@ -256,25 +263,23 @@ let Map = class Map extends React.Component {
         sourceLayer: "original",  
         layout: {},
         paint: {
-          /*['case',
-          ['boolean', ['feature-state', 'click'], false],
-          '#696969',
-  
-          ]*/
           "fill-color": {
             property: this.props.active.property,
             stops: this.props.active.stops,
           },
-          "fill-opacity": [
-            "case",
-            ["boolean", ["feature-state", "click"], false],
-            1,
-            ["boolean", ["feature-state", "highlight"], false],
-            1,
-            ["boolean", ["feature-state", "hover"], false],
-            1,
-            0.8,
-          ],
+          "fill-opacity": 0.8,
+        },
+      });
+      this.map.addLayer({
+        id: "sa2-borders",
```

Comment:
`Error: Layer with id "sa2-borders" already exists on this map`

I actually don't get any borders on the minimap unless I rename this, because the borders from rest of the features already use that layer id. You can change sa2-borders here to anything else though and it works.